### PR TITLE
Bank Management Project

### PR DESCRIPTION
**Pull request by PUNEETMAURYA**
Created using C++ ; Running Perfectly ; Minor Project ;Hacktoberfest2020 (please approve)

### CODE REVIEW

**Comment by chandrikadeb7** [comment]
@PUNEETMAURYA Please enter your file in correct folder and follow the file structure.

**Review by nimishbongale — COMMENTED** [review]
@PUNEETMAURYA Good job!

However, you must follow the folder structure and file nomenclature while contributing, as @chandrikadeb7 rightly mentioned. Let's get this merged!

**Comment by PUNEETMAURYA** [comment]
@nimishbongale  @chandrikadeb7  Thanks for appreciation , but I don't know the correct file structure for you repo. Do I need to keep this in a folder or it is  something related to renaming  of the project in certain format? Please tell the correct file nomenclature and folder structure.

File: bank_management.cpp [hunk 1]
```diff
@@ -0,0 +1,115 @@
+// C++ program to implement
```
Comment (nimishbongale) [inline bank_management.cpp:1]: You need to have this file put under the `Task 3` folder. That's about it!

Comment (PUNEETMAURYA) [reply bank_management.cpp:1]: @nimishbongale  thanks, I have reuploaded with correct structure . Please consider.

Comment (nimishbongale) [inline bank_management.cpp:1]: Please delete this file now that you have moved it to the right location.

**Review by nimishbongale — COMMENTED** [review]
Good work @PUNEETMAURYA!

On closer inspection, I personally think that this program should lie in the task 1 folder with the `question.md` file attached with it. Task 3 is only for algorithms implemented in certain languages.

**Pull request closed without merging**